lib/flatpak-installation.c

```diff
@@ -716,7 +738,7 @@ flatpak_installation_list_installed_refs (FlatpakInstallation *self,
                                           GCancellable        *cancellable,
                                           GError             **error)
 {
-  g_autoptr(FlatpakDir) dir = flatpak_installation_get_dir (self);
+  g_autoptr(FlatpakDir) dir = flatpak_installation_get_dir_maybe_no_repo (self);
```

Comment:
I think list_installed_refs_by_kind() should be maybe_no_repo too.

Reply:
Fixed, and also made get_id(), get_display_name(), and get_storage_type() use maybe_no_repo as well -- does that make sense to you?

Reply:
Yeah, that looks good.